src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -694,6 +705,18 @@ protected bool IsAlive
         }
 
         #region Location extraction & manipulation
+        /// <summary>
+        /// Sets an exsisting element's pinned status
+        /// </summary>
+        /// <param name="value">Value for pin status true/false</param>
+        public void SetPinnedStatus(bool value)
```

Comment:
is this in the right `region` ? seems to be in `Location extraction & manipulation` whereas the other pinned node is on line 157

Reply:
that's a mistake with the region! 
The other pinned node is a property so I placed it with the other properties, should we group all new nodes in regions?

Reply:
the `IsPinned` is in the right place, just had a look at the entire file, maybe let's put `SetPinnedStatus` before the regions begin ? 
https://github.com/SHKnudsen/DynamoRevit/blob/c44173ba4067b764790f567c915c6ed27014baff/src/Libraries/RevitNodes/Elements/Element.cs#L514